autogen/agentchat/conversable_agent.py

```diff
@@ -143,6 +143,11 @@ def __init__(
             self.llm_config = self.DEFAULT_CONFIG.copy()
             if isinstance(llm_config, dict):
                 self.llm_config.update(llm_config)
+            # We still have a default `llm_config` because the user didn't
+            # specify anything. This won't work, so raise an error to avoid
+            # an obscure message from the OpenAI service.
+            if self.llm_config == self.DEFAULT_CONFIG:
+                raise ValueError("Please specify the value for 'llm_config'.")
```

Comment:
It's not always necessary to raise. If the DEFAULT_CONFIG is modified by the developer to be a valid llm_config, then we shouldn't raise.

Reply:
OK, will explicitly check against `{}` to make sure that there's configuration.